procrastinate/sqlalchemy_connector.py

```diff
@@ -0,0 +1,176 @@
+import functools
```

Comment:
Should this be in contrib ?

Reply:
Yes, could be. I have no problem with moving it to the contrib folder. But do we agree that it's just a folder change, and that contribs have nothing else specific?

Reply:
I moved the connector to the `contrib.sqlalchemy` module.